### DISCUSSION TITLE
Logo for project

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/RepoRater#30] [Ideas]
I have subscribe to Envato Elements, so we can use a logo from there

You can search and find logos you like and share below. 

Here is an example https://elements.envato.com/graphic-templates/logos/rate

**Comment by eddiejaoude** [comment]
This one looks nice, what do you think?  https://elements.envato.com/seo-gear-logo-HD6UYD

**Comment by kumarsonsoff3** [comment]
What do you'll think of this one: https://elements.envato.com/stars-house-gradient-logo-design-EQ9ACRK

**Reply by eddiejaoude** [reply]
Looks good, but looks a bit like a house rating

**Reply by kumarsonsoff3** [reply]
I agreed, had some other ones customized using https://app.logo.com/
![image](https://github.com/EddieHubCommunity/RepoRater/assets/88786642/46a705b2-7c6a-4cc5-8446-dae357953502)

**Comment by CBID2** [comment]
Here’s this one: https://elements.envato.com/logo-star-award-winner-leader-ribbon-design-style-CLE8XY3

**Reply by eddiejaoude** [reply]
Looks good, but looks a bit Christmasy

**Comment by eddiejaoude** [comment]
https://elements.envato.com/personal-stats-logo-template-NPPNX87

**Comment by eddiejaoude** [comment]
https://elements.envato.com/chat-stats-logo-KYHLBRW

**Comment by eddiejaoude** [comment]
https://elements.envato.com/seo-gear-logo-HD6UYD

**Comment by eddiejaoude** [comment]
https://elements.envato.com/analytics-chat-logo-QF296F

**Comment by eddiejaoude** [comment]
https://elements.envato.com/computer-seo-logo-MUU2XV3

**Comment by eddiejaoude** [comment]
https://elements.envato.com/code-discussion-logo-VLNQZY

**Comment by eddiejaoude** [comment]
https://elements.envato.com/code-geek-logo-CL9SYT

**Comment by eddiejaoude** [comment]
![Asset 18000](https://github.com/EddieHubCommunity/RepoRater/assets/624760/467d4052-cd4e-49fb-9617-b5b13de2bdbd)
